src/vm/stackingallocator.h

```diff
@@ -196,6 +196,26 @@ class StackingAllocator
 
     void* UnsafeAllocSafeThrow(UINT32 size);
     void* UnsafeAlloc(UINT32 size);
+    void ClearUnallocated()
```

Comment:
Should we unify this with the JIT slab allocator (JitHost::allocateSlab) ?

Reply:
No, I don't think so. The JIT slab allocator is working with much larger blocks of memory and needs cross-thread operations to cleanup. Of course, we could allocate a slab at type load time start, which would probably be ok, but that's not really any better than this approach, and will require more memory overall. This approach is better for the shorter lifetime transient allocations of the typeloader (by a small margin).